src/Http/Response.php

```diff
@@ -2233,6 +2233,21 @@ public function withFile($path, array $options = [])
     }
 
     /**
+     * Convenience method to set a string into the response body
+     *
+     * @param string $string The string to be sent
+     * @return static
+     */
+    public function withStringBody($string)
+    {
+        $new = clone $this;
+        $new->_createStream();
```

Comment:
Yeah, why wouldn't that work? It's a subclass calling a `private` method that won't work.